mlflow/spark/__init__.py

```diff
@@ -167,6 +167,13 @@ def log_model(
         spark_model: Spark model to be saved - MLflow can only save descendants of
             pyspark.ml.Model or pyspark.ml.Transformer which implement
             MLReadable and MLWritable.
+
+                .. Note:: The provided Spark model `transform` method must generate one column
+                    named with "prediction", the column is used as MLflow pyfunc model output.
+                    Most Spark models set "prediction" column as the output label column
```

Comment:
```suggestion
                    Most Spark models set "prediction" as the output column label
```
Since "prediction" is the column name :)

Reply:
“the output column label” sounds ambiguous, what about "the output column containing model prediction labels"

Reply:
Rewording:

```
Most Spark models generate the output column with "prediction" name that
                    contains prediction labels.
```

Reply:
Do we need this sentence? Maybe we can delete it

Reply:
I think we should inform user that the default "prediction" column contains prediction labels , not other things like probabilities or other data structure